{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:39.597Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:39.606Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:39.749Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:40.736Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:40.740Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:41.736Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:41.738Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:42.699Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:43.261Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:43.601Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:44.215Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:44.404Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:44.706Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:45.296Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:20:46.150Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:16.516Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:16.643Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:16.944Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:17.708Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:19.736Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:24.586Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:24.821Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:24.897Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:25.077Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:890:11)\n    at exports._exceptionWithHostPort (util.js:913:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)","timestamp":"2016-05-18T09:22:26.194Z"}
